contributing_to_docs/doc_guidelines.adoc

```diff
@@ -334,28 +314,49 @@ The `prerequisites_`, `additional-resources_`, and `next-steps_` prefixes must e
 
 The following examples include IDs that are unique to the "Configuring alert notifications" assembly:
 
-*Example unique ID for a "Prerequisites" title*
-
+.Example unique ID for a "Prerequisites" title
 ----
 [id="prerequisites_configuring-alert-notifications"]
 == Prerequisites
 ----
 
-*Example unique ID for an "Additional resources" title*
-
+.Example unique ID for an "Additional resources" title
 ----
 [role="_additional-resources"]
 [id="additional-resources_configuring-alert-notifications"]
 == Additional resources
 ----
 
-*Example unique ID for a "Next steps" title*
-
+.Example unique ID for a "Next steps" title
 ----
 [id="next-steps_configuring-alert-notifications"]
 == Next steps
 ----
 
+=== Discrete headings
+
+If you have a section heading that you do not want to appear in the TOC (like if you think that some section is not worth showing up or if there are already too many nested levels), you can use a discrete (or floating) heading:
```

Comment:
```suggestion
If you have a section heading that you do not want to appear in the TOC (for example, if you think that some section is not worth showing up or if there are already too many nested levels), you can use a discrete (or floating) heading:
```

Reply:
I don't think this necessarily needs to be changed just a suggestion to get rid of `like` here.